New package: ORTools v0.0.1

JuliaRegistrator opened the pull request:
- Registering package: ORTools
- Repository: https://github.com/google/or-tools
- Created by: @dourouc05
- Version: v0.0.1
- Commit: 1325fdc1c7d50c1aee16733bfedc085586518912
- Reviewed by: @dourouc05
- Reference: https://github.com/google/or-tools/commit/1325fdc1c7d50c1aee16733bfedc085586518912#commitcomment-162556354
- Description: Google's Operations Research tools:
<!-- ae9404edfd72f1de901db3e818bff6d68e1033f2a40e4f25e4af6c579aabbd894258bc4fdfc87f7f74551bbe06810b12135fc24bc7712ae0d906745d069ace0eedf5d13a12a281e9bb1b8c8f892ae7ab481dbe06a5ffa188ad549c0bbf99ee6e1218c929a55e833af9ff28ecba6e6b3d26b94d473f81d6d77b056fe3afbb0f864119bbc1e18f6d4026b34afad2c800b8107358490d735029ae2890cac5bc93ecbcd6cdc5eec1720db3f4794b778421f36f6ecb548b5f433b3d23066e4ba88f6b41f31bc4f5eccf534ec030b1b70ca5f953bc94cffc2f17a67ae8d5925e4ba329a6f1fb756889f5b03673a8844ec985b3 -->

dourouc05 commented:
Most errors should be solved when https://github.com/JuliaRegistries/General/pull/135128 is in. 

Regarding the names: this package provides a interface to the library OR-Tools, it's not like we have many choices here :).

JuliaRegistrator committed:
New package: ORTools v0.0.1

UUID: b7d69b34-a827-4671-8cfa-f7e1eec930c7
Repo: https://github.com/google/or-tools.git
Tree: 732e25dcd6c9af81d6172989f56d7c9c48212d3a

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
